src/utils/theme.js

```diff
@@ -18,18 +19,25 @@ export let mergeNestedTheme = (theme, key) =>
     )
   )(theme)
 
-export let ThemeConsumer = ({ name, children, theme: propTheme }) => {
+let mergeThemes = (name, propTheme) => {
```

Comment:
Since this is effectively a custom hook, let's follow the convention and call it `useTheme`:

```js
let useTheme = (name, theme) =>
  mergeOrReturn(mergeNestedTheme(React.useContext(ThemeContext), name), theme)
```